### DISCUSSION TITLE
Sort by date under Search

### DISCUSSION THREAD

**Discussion by selvarajsubramanian** [dailydotdev/daily#430] [Feature Requests]
Hi Team,
I feel it will be helpful if we have sort by date feature in the Search operation. Thanks much for adding the Sort by date option in the Popular feed recently.

Thanks,
Selva

**Comment by idoshamun** [comment]
Hi @selvarajsubramanian,

Thanks for the suggestion. We have it on our roadmap actually. We want to add advanced options to the search.
Stay tuned, we will be sure to update it in the changelog.

**Reply by selvarajsubramanian** [reply]
Great, Thank you!

**Reply by idoshamun** [reply]
Let's keep the convo here: https://github.com/dailydotdev/daily/discussions/294
